app/models/AuthUserSchema/AuthUserSchema.js

```diff
@@ -0,0 +1,44 @@
+import mongoose from "mongoose";
+import bcrypt from "bcrypt";
+import { validateEmail } from "../validators";
+
+const SALT_WORK_FACTOR = 10;
+
+export const AuthUserSchema = new Schema({
+  login: {
+    type: String,
+    lowercase: true,
+    trim: true,
+    required: true,
+    unique: true,
+  },
+  email: {
+    type: String,
+    lowercase: true,
```

Comment:
Same here. Is there any reason for e-mail to be lowercase?

Reply:
same up